templates/stripes/stripe-header-video.php

```diff
@@ -20,24 +29,35 @@
         </div>
         <?php endif; ?>
 
-        <?php if (!empty($inset) || !empty($bgVideoUrl)) : ?>
+        <?php if (!empty($inset) || !empty($bgVideoUrl)): ?>
         <div class="vssl-stripe--header-video--inset">
             <div class="vssl-stripe--header-video--ui">
-                <?php if (!empty($bgVideoUrl)) : ?>
+                <?php if (!empty($bgVideoUrl)): ?>
                 <video loop autoplay muted playsinline
-                    <?= !empty($inset) ? 'poster="' . $this->image($inset, $image_style ?? null) . '"' : '' ?>
+                    <?= !empty($inset)
+                      ? 'poster="' .
+                        $this->image($inset, $image_style ?? null) .
+                        '"'
+                      : "" ?>
                 >
                     <source src="<?= $bgVideoUrl ?>" type="video/mp4">
-                    <?php if (!empty($inset)) : ?>
-                    <img src="<?= $this->image($inset, $image_style ?? null) ?>" alt="">
+                    <?php if (!empty($inset)): ?>
+                    <img src="<?= $this->image(
+                      $inset,
+                      $image_style ?? null
+                    ) ?>" alt="<?= $inset_alt ?? "" ?>"/>
                     <?php endif; ?>
                 </video>
-                <?php else : ?>
-                <img src="<?= $this->image($inset, $image_style ?? null) ?>" alt="">
+                <?php else: ?>
+                <img src="<?= $this->image(
+                  $inset,
+                  $image_style ?? null
+                ) ?>" alt="<?= $inset_alt ?? "" ?>"/>
```

Comment:
here and in the `<img>` tag above, using a new `$inset_alt` property for the alt attribute